shopping_mall/backend/ai/agent/executor.py

```
@@ -0,0 +1,652 @@
+
+            skip_summary = ", ".join(skipped)
+            return (
+                f"\n도착예정: {adjusted.strftime('%Y-%m-%d')} "
+                f"(원래 {arrival_date.strftime('%Y-%m-%d')}이었으나 {skip_summary} 제외하여 조정)"
+            )
+        except Exception as e:
+            logger.warning(f"영업일 조정 실패: {e}")
+            return f"\n도착예정: {arrival_date.strftime('%Y-%m-%d')}"
+
+    async def _tool_search_products(
+        self, db: Session, query: str, check_stock: bool = False, limit: int = 5
```

Ruff (0.15.9)

[warning] Boolean-typed positional argument in function definition

(FBT001)